.eslintrc.cjs

```diff
@@ -4,7 +4,7 @@ module.exports = {
     es2021: true,
   },
   root: true,
-  extends: ['standard-with-typescript', 'plugin:react/recommended'],
```

Comment:
This plugin only appears to be used in app, so configuring it here seems to be a mistake

Reply:
It was intentional I think, because it didn't harm anything, but can't remember much beyond that